python/ray/train/tests/test_backend.py

```diff
@@ -53,6 +56,20 @@ def ray_4_node_4_cpu():
     cluster.shutdown()
 
 
+@pytest.fixture
+def ray_2_node_2_cpu():
```

Comment:
can these be moved to conftest.py ?  All you need is to add `deps = [":train_lib", ":conftest"]` in the corresponding pytest build https://sourcegraph.com/github.com/ray-project/ray/-/blob/python/ray/train/BUILD

Reply:
This fixture is only needed for the two specific tests I have added. 
If we move this to conftest.py then shouldn't we also move other cluster setup fixtures to the conftest.py file?

Reply:
we had a bad pattern that redefined a bunch of fixtures in each test file in the past that some of them are being addressed already, if this fixture is only used in this file im fine living it as-is, but please keep this context in mind when you come across other fixtures like the ones above, we might still have a few hanging around that are defined more than once.

Reply:
Fixtures have been moved to conftest and conftest is added as a BUILD dependency